tests/functional/n/non/non_ascii_name.py

```diff
@@ -1,6 +1,6 @@
 """ Tests for non-ascii-name checker. """
 
-áéíóú = 4444 # [non-ascii-name]
+áéíóú = 4444 # [non-ascii-identifier]
```

Comment:
Could you rename these files? They no longer reflect the message name.

Reply:
> Got to the other files as well!
> 
> @CarliJoy Tests look very good and robust. tada I just wonder if we can move some of the remaining unittests to the functional tests as well. I'm asking this not only because I find them more readable, but also since I know they are actually better "tests". They way those are invoked better resembles the way `pylint` is invoked by normal users than the unittests are, which has recently caught a bug that the unittests simply could not in their current setup. This is something we as maintainers/contributors should fix, but in the meantime functional tests are often the way to go.

@DanielNoord which unit test are you referring to?
Because I added functional test for the most use cases. I escpecially use the unit tests for two things:
a) To check that actually got the correct node and make sure that I understood the logic correctly (internal logic checks)
b) To use parameterize to check a function for all possible outcomes. This would be very cumbersome to this manually in funcrtional tests, if there is not equivalent of parameterize. Some cases I use there are still tested within the functional tests. So testing all possiblites with unit test and a bunch of real world examples should actually cover everything IMO.

Reply:
Kept the original name for the message and freed me from renaming every file.... (was easier)